include/etl/priority_queue.h

```diff
@@ -306,17 +306,27 @@ namespace etl
     }
 
     //*************************************************************************
-    /// Removes the oldest value from the back of the priority queue.
+    /// Removes the highest priority value from the top of the priority queue.
     /// Does nothing if the priority queue is already empty.
     //*************************************************************************
     void pop()
     {
-      // Move largest element to end
+      // Move largest (highest priority) element to end
       etl::pop_heap(container.begin(), container.end(), compare);
       // Actually remove largest element at end
       container.pop_back();
     }
 
+    //*************************************************************************
```

Comment:
I've found it sometimes helpful to add an explicit comment to flag that these our changes if they aren't conforming to the upstream report, and in this case the std library version. Especially if we are unlikely to upstream these things.

Something like wrapping this `/*+++++++++ SPAN +++++++++*/` so it's obvious when looking at this code outside of git and other source control tools.